examples_gbs/run_tutorial_sample.py

```diff
@@ -0,0 +1,140 @@
+# pylint: disable=invalid-name,no-member,wrong-import-position,wrong-import-order,ungrouped-imports
+"""
+Sampling Tutorial
+=================
+
+This tutorial provides a quick guide to generating samples from GBS. This functionality is

```

Comment:
Should we mention explicitly that this employs simulators? e.g. "... generating samples from GBS simulators?"